<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sign_up_page</class>
 <widget class="QWidget" name="Sign_up_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>725</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="back_push">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>610</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sign_push">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>510</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>211</width>
     <height>77</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Last Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lastNmae_le">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>111</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Age:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="age_spin">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>120</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>211</width>
     <height>77</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm Password:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="confPassword_le">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>201</width>
     <height>77</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="username_le">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>421</width>
     <height>99</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Address:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="address_te">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>201</width>
     <height>77</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="password_le">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>201</width>
     <height>77</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Phone Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="phoneNumber_le">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>201</width>
     <height>77</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>First Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="firstNmae_le">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sign Up as:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Customer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Employee</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>471</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../images/images.qrc">:/signup_pic.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images/images.qrc"/>
  <include location="../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
